platform/cli/cli.ts

```diff
@@ -20,7 +21,7 @@ export const parseArgv = (argv: Array<string>) =>
     .epilogue(args.docs)
     .check(args.check(args.options)).argv;
 
-const run = (argv: { [key: string]: any }) => {
+export const run = (argv: { [key: string]: any }) => {
```

Comment:
It feels like we should create a separate `runWithLegacyKbnServer`. We probably need to explore not listening for `SIGHUP`, `SIGINT` etc here, and rather receive that through the old platform somehow. I've got a feeling we might want to receive the config from the old platform too, and have a `transformLegacyConfig` or something like that which transforms it into whatever we need/want in the new platform.

Reply:
(Or just rewrite this code to be explicit about `kbnServer` always being there, and getting rid of `node scripts/platform` entirely. Then, when we get to K7 we can add whatever we need to have the new platform start cleanly on its own)

Reply:
Don't have strong opinion on that, having `runWithLegacyKbnServer` sounds reasonable to me. I'd get rid of `node scritps/platform` only once we have dev setup ready, I still use `ts:start` with `scripts/platform` to quickly test changes.

> I've got a feeling we might want to receive the config from the old platform too, and have a transformLegacyConfig or something like that which transforms it into whatever we need/want in the new platform.

I'll explore that.